File: app/src/main/java/com/lazylee/lzywanandroid/adapter/viewholder/HotKeyViewHolder.java
package com.lazylee.lzywanandroid.adapter.viewholder;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;

import com.lazylee.lzywanandroid.R;
import com.lazylee.lzywanandroid.data.entity.HotKey;
import com.lazylee.lzywanandroid.view.LzyChip;


public class HotKeyViewHolder extends RecyclerView.ViewHolder {

    private static final String TAG = "HotKeyViewHolder";
    LzyChip chip;

    public HotKeyViewHolder(View itemView) {
        super(itemView);
        chip = itemView.findViewById(R.id.hot_key_chip);
    }

    public void bindView(HotKey hotKey) {
        chip.setChipText(hotKey.getName());
    }
}

File: app/src/main/java/com/lazylee/lzywanandroid/ui/adapter/viewholder/HotKeyViewHolder.kt
package com.lazylee.lzywanandroid.ui.adapter.viewholder


import androidx.recyclerview.widget.RecyclerView
import android.view.View

import com.lazylee.lzywanandroid.R
import com.lazylee.lzywanandroid.data.entity.HotKey
import com.lazylee.lzywanandroid.ui.view.LzyChip


class HotKeyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    internal var chip: LzyChip

    init {
        chip = itemView.findViewById(R.id.hot_key_chip)
    }

    fun bindView(hotKey: HotKey) {
        chip.setChipText(hotKey.name)
    }

    companion object {

        private val TAG = "HotKeyViewHolder"
    }
}
